### PR TITLE
[formrecognizer] more multipage form test coverage

### LINKED ISSUE #11713: [formrecognizer] Increase test sample coverage and ensure test parity across languages
This issues tracks 2 things we'd like to accomplish before GA:

1) We need more sample forms from the service team so that we are testing many types of forms (especially multipage ones). We have reached out to the service team and the sample gap table has been shared and posted in this issue: https://github.com/Azure/azure-sdk-for-net/issues/12139

2) Sometimes we see live tests failures from one language, but not from another. We should focus on having test parity across languages where it makes sense. A spreadsheet has been posted to the FR Teams Channel and all languages will document their current test scenarios. Once finished, we will add tests to our test suite where other languages have that test and it is not found in python.

### PR DESCRIPTION
**Pull request by kristapratico**
Resolves #11713

### CODE REVIEW

File: sdk/formrecognizer/azure-ai-formrecognizer/tests/test_content_from_url.py [hunk 1]
```diff
@@ -165,4 +165,52 @@ def test_content_continuation_token(self, resource_group, location, form_recogni
...
+        client = FormRecognizerClient(form_recognizer_account,
+                                      AzureKeyCredential(form_recognizer_account_key))
+
+        poller = client.begin_recognize_content_from_url(self.multipage_table_url_pdf)
+        result = poller.result()
+        self.assertEqual(len(result), 2)
+        layout = result[0]
+        self.assertEqual(layout.page_number, 1)
+        self.assertEqual(len(layout.tables), 2)
+        self.assertEqual(layout.tables[0].row_count, 30)
```
Comment (iscai-msft) [inline sdk/formrecognizer/azure-ai-formrecognizer/tests/test_content_from_url.py:181]: Should we also check lines?

File: sdk/formrecognizer/azure-ai-formrecognizer/tests/test_content_from_url.py [hunk 2]
```diff
@@ -165,4 +165,52 @@ def test_content_continuation_token(self, resource_group, location, form_recogni
...
+        self.assertEqual(layout.tables[0].column_count, 5)
+        self.assertEqual(layout.tables[0].page_number, 1)
+        self.assertEqual(layout.tables[1].row_count, 6)
+        self.assertEqual(layout.tables[1].column_count, 5)
```
Comment (iscai-msft) [inline sdk/formrecognizer/azure-ai-formrecognizer/tests/test_content_from_url.py:185]: also would it be better to have a helper function that compares two form pages

File: sdk/formrecognizer/azure-ai-formrecognizer/tests/test_content_from_url.py (hunk 1 above)
Comment (kristapratico) [reply sdk/formrecognizer/azure-ai-formrecognizer/tests/test_content_from_url.py:181]: Done in the test that checks the result transform

File: sdk/formrecognizer/azure-ai-formrecognizer/tests/test_content_from_url.py (hunk 2 above)
Comment (kristapratico) [reply sdk/formrecognizer/azure-ai-formrecognizer/tests/test_content_from_url.py:185]: done in the transform test

**Pull request merged**